scripts/update_robots.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+#------------------------------------------------------------------------------
+# Prints version of the Solidity compiler that the source code corresponds to.
+#
+# The documentation for solidity is hosted at:
+#
+#     https://docs.soliditylang.org
+#
+# ------------------------------------------------------------------------------
+# This file is part of solidity.
+#
+# solidity is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# solidity is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with solidity.  If not, see <http://www.gnu.org/licenses/>
+#
+# (c) 2017 solidity contributors.
+#------------------------------------------------------------------------------
+
+set -e
+set -u
+
+PATH_TO_GET_VERSION_SCRIPT="./scripts/get_version.sh"
+PATH_TO_ROBOTS_TXT_FILE="docs/_static/robots.txt"
+TEMP_FILE="temp_store_robots.txt"
+VERSION_FROM_SCRIPT=`bash ${PATH_TO_GET_VERSION_SCRIPT} | perl -pe 'chomp'`
+LAST_LINE_ROBOTS=`tail -n 1 ${PATH_TO_ROBOTS_TXT_FILE}`
+VERSION_NUMBER_IN_ROBOTS="#\w+:\s\/([a-z]+)\/v([0-9]\.[0-9]\.[0-9])\/"
+[[ ${LAST_LINE_ROBOTS} =~ ${VERSION_NUMBER_IN_ROBOTS} ]]
+if [[ $? == 0 ]]; then
+    if [[ ${BASH_REMATCH[2]} != ${VERSION_FROM_SCRIPT} ]]; then
```

Comment:
Last warning from [`chk_coding_style` job in CI](https://app.circleci.com/pipelines/github/ethereum/solidity/14768/workflows/17530589-877e-4e39-95d8-2fd4e075dc4b/jobs/675064):

```
In ./scripts/update_robots.sh line 40:
    if [[ ${BASH_REMATCH[2]} != ${VERSION_FROM_SCRIPT} ]]; then
                                ^--------------------^ SC2053: Quote the right-hand side of != in [[ ]] to prevent glob matching.
```

You're not quoting variables properly.

Also, you do not need the `${}` around every variable. It's only needed when the variable is surrounded by text, e.g. `echo "XXX${value}XXX"`. I generally use it when I use a variable in a string but I think that it just adds noise when the variable stands on its own.

```suggestion
    if [[ $BASH_REMATCH[2] != "$VERSION_FROM_SCRIPT" ]]; then
```